### ISSUE TITLE
Heroku Deploy Error

### ISSUE THREAD

**Issue by MAkcanca** [mirumee/saleor#1521]
Hi!
Whenever I use heroku deploy button, I receive following error on step 4, run scripts & scale dynos.

> Shipping method #1
Shipping method #2
INFO elasticsearch POST https://birch-2016774.eu-west-1.bonsaisearch.net:443/_bulk?refresh=true [status:200 request:0.054s] [PID:11:MainThread]
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/app/.heroku/python/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/app/saleor/core/management/commands/populatedb.py", line 58, in handle
    stdout=self.stdout)
  File "/app/saleor/core/utils/random_data.py", line 245, in create_products_by_schema
    how_many=how_many, create_images=create_images, stdout=stdout)
  File "/app/saleor/core/utils/random_data.py", line 209, in create_products_by_class
    product = create_product(product_class=product_class)
  File "/app/saleor/core/utils/random_data.py", line 287, in create_product
    return Product.objects.create(**defaults)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/db/models/base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/db/models/base.py", line 847, in save_base
    update_fields=update_fields, raw=raw, using=using,
  File "/app/.heroku/python/lib/python3.6/site-packages/django/dispatch/dispatcher.py", line 193, in send
    for receiver in self._live_receivers(sender)
  File "/app/.heroku/python/lib/python3.6/site-packages/django/dispatch/dispatcher.py", line 193, in <listcomp>
    for receiver in self._live_receivers(sender)
  File "/app/.heroku/python/lib/python3.6/site-packages/django_elasticsearch_dsl/signals.py", line 51, in handle_save
    registry.update(instance)
  File "/app/.heroku/python/lib/python3.6/site-packages/django_elasticsearch_dsl/registries.py", line 40, in update
    doc().update(instance, **kwargs)
  File "/app/.heroku/python/lib/python3.6/site-packages/django_elasticsearch_dsl/documents.py", line 216, in update
    self._get_actions(object_list, action), **kwargs
  File "/app/.heroku/python/lib/python3.6/site-packages/django_elasticsearch_dsl/documents.py", line 176, in bulk
    return bulk(client=self.connection, actions=actions, **kwargs)
  File "/app/.heroku/python/lib/python3.6/site-packages/elasticsearch/helpers/__init__.py", line 257, in bulk
    for ok, item in streaming_bulk(client, actions, **kwargs):
  File "/app/.heroku/python/lib/python3.6/site-packages/elasticsearch/helpers/__init__.py", line 192, in streaming_bulk
    raise_on_error, **kwargs)
  File "/app/.heroku/python/lib/python3.6/site-packages/elasticsearch/helpers/__init__.py", line 137, in _process_bulk_chunk
    raise BulkIndexError('%i document(s) failed to index.' % len(errors), errors)
elasticsearch.helpers.BulkIndexError: ('1 document(s) failed to index.', [{'index': {'_index': 'storefront', '_type': 'product_document', '_id': '1', 'status': 404, 'error': {'type': 'index_not_found_exception', 'reason': "no such index and [action.auto_create_index] ([*logstash*,*requests*,*events*,*.kibana*,*kibana-int*,*filebeat*]) doesn't match", 'index_uuid': '_na_', 'index': 'storefront'}, 'data': {'title': 'Day-Santana', 'name': 'Day-Santana', 'description': 'Harum et aliquam quo nemo aliquam voluptates magni. Incidunt eligendi adipisci temporibus magni adipisci. Consequuntur laudantium ut numquam soluta nisi possimus aut. Consectetur veniam ab est culpa unde amet.\n\nPossimus voluptatem necessitatibus magnam voluptatem provident laudantium dolore perferendis. Aliquid nostrum cupiditate earum cupiditate qui. Quas dolorum tempore magni recusandae ducimus.\n\nQuod fuga occaecati earum hic architecto rerum. Quae magnam repellendus placeat alias dicta vero. Repudiandae sed sequi velit voluptatum repudiandae itaque reiciendis.\n\nSimilique modi pariatur perspiciatis molestiae corrupti quod. Deleniti enim autem suscipit tempore. Qui pariatur repudiandae culpa maiores quisquam.\n\nProvident sint consectetur doloribus sint. Voluptas nemo enim suscipit consequuntur quo quaerat eligendi. Ipsa amet aliquid quaerat officia eligendi maxime.', 'is_published': True}}}])

**Comment by MAkcanca** [comment]
With following config,
[https://preview.ibb.co/nw6gs6/image.png](url)

**Comment by MAkcanca** [comment]
I guess you guys are tired of Heroku

**Comment by TheCuriousLearner** [comment]
Same problem here as well.

**Comment by hoaluvn** [comment]
Same problem

**Comment by jamespsteinberg** [comment]
same problem

**Comment by jamespsteinberg** [comment]
@raodienne this solves it ^

**Comment by MAkcanca** [comment]
@jamespsteinberg Yes, it could but we are talking about the deploy button on Readme page of this repo :)

**Comment by TotesMagoats** [comment]
Same problem here.  I thought it would be fun to deploy quickly and play around with it but turns out the deploy to Heroku button doesn't work.

**Comment by elwoodxblues** [comment]
The issue should be resolved by PR #1785.

**Comment by khashashin** [comment]
I have now the same issue, this command `heroku run python manage.py search_index --create` throws following error: 
```
Creating index 'orders'
WARNING elasticsearch PUT https://ivy-75841735.eu-west-1.bonsaisearch.net:443/orders [status:400 request:0.042s] [PID:4:MainThread]
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/app/.heroku/python/lib/python3.7/site-packages/django_elasticsearch_dsl/management/commands/search_index.py", line 128, in handle
    self._create(models, options)
  File "/app/.heroku/python/lib/python3.7/site-packages/django_elasticsearch_dsl/management/commands/search_index.py", line 84, in _create
    index.create()
  File "/app/.heroku/python/lib/python3.7/site-packages/elasticsearch_dsl/index.py", line 203, in create
    self.connection.indices.create(index=self._name, body=self.to_dict(), **kwargs)
  File "/app/.heroku/python/lib/python3.7/site-packages/elasticsearch/client/utils.py", line 76, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/app/.heroku/python/lib/python3.7/site-packages/elasticsearch/client/indices.py", line 88, in create
    params=params, body=body)
  File "/app/.heroku/python/lib/python3.7/site-packages/elasticsearch/transport.py", line 318, in perform_request
    status, headers_response, data = connection.perform_request(method, url, params, body, headers=headers, ignore=ignore, timeout=timeout)
  File "/app/.heroku/python/lib/python3.7/site-packages/elasticsearch/connection/http_urllib3.py", line 186, in perform_request
    self._raise_error(response.status, raw_data)
  File "/app/.heroku/python/lib/python3.7/site-packages/elasticsearch/connection/base.py", line 125, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
elasticsearch.exceptions.RequestError: RequestError(400, 'mapper_parsing_exception', 'Root mapping definition has unsupported parameters:  [doc : {properties={user_email={type=text}, discount_name={type=text}, last_name={type=text}, user={analyzer=email_analyzer, type=text}, first_name={type=text}}}]')
(env)
```
